Validation/GlobalHits/interface/GlobalHitsAnalyzer.h

```diff
@@ -59,7 +59,7 @@
 
 // helper files
 //#include <CLHEP/Vector/LorentzVector.h>
-#include "CLHEP/Units/GlobalSystemOfUnits.h"
+#include "CLHEP/Units/SystemOfUnits.h"
```

Comment:
Here also <CLHEP/Units/SystemOfUnits.h> is recommended.

Reply:
ah, is your comment about `<` instead of `"` ?